content/guides/component-testing/mounting-angular.md

```diff
@@ -0,0 +1,126 @@
+
+<code-group>
+<code-block label="cypress/support/component.ts" active>
+
+```ts
+import { mount } from 'cypress/angular'
+
+Cypress.Commands.add('mount', mount)
+```
+
+</code-block>
+</code-group>
+
+This allows you to use `cy.mount()` in any component test without having to
+import the framework-specific mount command.
+
+### Your First Component Test
+
+Now that you have a component and know how to mount it let's write our first
+test.
+
+To get started, create a spec file in the same directory as the
+`stepper.component.ts` component and name it **stepper.component.cy.ts**. Then
+paste the following into it:
+
+<code-group>
+<code-block label="stepper.component.cy.ts" active>
+
+```ts
+import { StepperComponent } from './stepper.component'
+
+describe('StepperComponent', () => {
+  it('mounts', () => {
+    cy.mount(StepperComponent)
+  })
+})
+```
+
+</code-block>
+</code-group>
+
+Here, we have a single test that ensures that our component mounts.
+
+## Running the Test
+
+Now it's time to see the test in action. Open up Cypress if it is not already
+running:
+
+```bash
+npx cypress open --component
+```
+
+> The `--component` flag will launch us directly into component testing
+
+And launch the browser of your choice. In the spec list, click on
+**stepper.component.cy.ts** and you will see the stepper component mounted in
+the test area.
+
+<DocsImage 
+  src="/img/guides/component-testing/first-test-run-angular.png" 
+  caption="Stepper Mount Test"> </DocsImage>
+
+A basic test that mounts a component in its default state is an excellent way to
+start testing. Since Cypress renders your component in a real browser, you have
+a lot of advantages, such as seeing that the component renders as it should,
+interacting with the component in the test runner, and using the browser dev
+tools to inspect and debug both your tests and the component's code.
+
+Feel free to play around with the `StepperComponent` by interacting with the
+increment and decrement buttons.
+
+### Configuring Your Component
+
+Angular applications are modular by nature. In fact, Angular has its own
+modularity system called `NgModules`. NgModules are containers for a cohesive
+block of code dedicated to an application domain, workflow, or a closely related
+set of capability. They can contain components, service providers, and other
+code files whose scope is defined by the containing NgModule. To summarize, an
+NgModule provides a _compilation context_ for it's components. In order to
+`mount` your component under test with cypress, you will use the
+`MountConfig<T>`, which is an interface that extends
+[TestModuleMetadata](https://angular.io/api/core/testing/TestModuleMetadata).
+You will find mounting your components with Cypress will feel very ergonomically
+angular.
+
+### Optional Template Support
+
+The mount command supports for Component Class syntax, but it can also be used
+with angular's template syntax _ie:_ `<app-button></app-button>`
```

Comment:
```suggestion
with Angular's template syntax _ie:_ `<app-button></app-button>`
```

Reply:
I think having a code block here showing how to use template syntax will help better explain the differences. You could have something like:

For instance, we could mount the stepper component from above using the template syntax like so:

```ts
import { StepperComponent } from './stepper.component'

describe('StepperComponent', () => {
  it('mounts', () => {
    cy.mount(`<app-stepper></app-stepper>)
  })
})
```

Using the template syntax allows use to write up inputs and outputs in a more succinct way and feels more natural in Angular. The rest of this guide will use template syntax when mounting Angular components.